lms/djangoapps/grades/tasks.py

```diff
@@ -115,6 +115,16 @@ def recalculate_subsection_grade_v3(self, **kwargs):
     _recalculate_subsection_grade(self, **kwargs)
 
 
+@task()
+def add(x, y):
+    return x + y
+
+
+@task
+def tsum(x):
+    return sum(x)
```

Comment:
For completion's sake - here's a similar execution with groups:

```
edxapp@vagrant:~/edx-platform$ ./manage.py lms --settings=devstack_with_worker shell
Python 2.7.12 (default, Nov 19 2016, 06:48:10) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
(InteractiveConsole)
>>> from lms.djangoapps.grades.tasks import add, tsum; from celery import group
>>> group(add.s(i, i) for i in xrange(10))()
<GroupResult: e5c64b1e-b8a5-4eb5-8113-55ee617c686b [bfc83fa0-c368-4f20-92ba-754de8241adf, 72313242-93fe-444d-9a3c-093b692ee830, 111ba008-7d88-496f-af25-8b97e35662ca, 94af9607-7fbf-4829-b854-a970469632e5, c60f0445-2b36-4be1-ae08-ff560ae0bba0, b0427bc7-55b8-4f4e-afda-b515eb80429d, 33744146-f76c-40d0-ab55-bcb759ef1ace, b25b9f07-679c-493b-9fa7-3ea8514b1d37, 261d0989-62d3-4440-ba52-d2793e777a2b, b94d1c1b-98a1-4b46-80b1-289ae5d246cb]>
```

Worker shell:
```
2017-04-19 20:49:33,072 INFO 13394 [celery.worker.strategy] strategy.py:55 - Received task: lms.djangoapps.grades.tasks.add[5a8fa272-2c7e-4b68-8698-aae6e8dffc4f]
2017-04-19 20:49:33,074 INFO 13394 [celery.worker.job] job.py:401 - Task lms.djangoapps.grades.tasks.add[5a8fa272-2c7e-4b68-8698-aae6e8dffc4f] succeeded in 0.000590919997194s: 0
2017-04-19 20:49:33,079 INFO 13394 [celery.worker.strategy] strategy.py:55 - Received task: lms.djangoapps.grades.tasks.add[55c6008f-8627-4210-8e56-6be8291a0793]
2017-04-19 20:49:33,080 INFO 13394 [celery.worker.strategy] strategy.py:55 - Received task: lms.djangoapps.grades.tasks.add[4b594539-177c-4a0e-8096-6441e1206456]
2017-04-19 20:49:33,084 INFO 13394 [celery.worker.job] job.py:401 - Task lms.djangoapps.grades.tasks.add[4b594539-177c-4a0e-8096-6441e1206456] succeeded in 0.0015249769931s: 4
2017-04-19 20:49:33,085 INFO 13394 [celery.worker.strategy] strategy.py:55 - Received task: lms.djangoapps.grades.tasks.add[792a8ea1-cd88-4d34-bfd6-adb61c0c8db6]
2017-04-19 20:49:33,086 INFO 13394 [celery.worker.job] job.py:401 - Task lms.djangoapps.grades.tasks.add[55c6008f-8627-4210-8e56-6be8291a0793] succeeded in 0.00410808599554s: 2
2017-04-19 20:49:33,087 INFO 13394 [celery.worker.strategy] strategy.py:55 - Received task: lms.djangoapps.grades.tasks.add[f9469ff6-7c19-47e0-8ccf-47c72a01b7ff]
2017-04-19 20:49:33,099 INFO 13394 [celery.worker.job] job.py:401 - Task lms.djangoapps.grades.tasks.add[f9469ff6-7c19-47e0-8ccf-47c72a01b7ff] succeeded in 0.00983558199368s: 8
2017-04-19 20:49:33,100 INFO 13394 [celery.worker.job] job.py:401 - Task lms.djangoapps.grades.tasks.add[792a8ea1-cd88-4d34-bfd6-adb61c0c8db6] succeeded in 0.011387342005s: 6
2017-04-19 20:49:33,101 INFO 13394 [celery.worker.strategy] strategy.py:55 - Received task: lms.djangoapps.grades.tasks.add[d7248e00-4aee-4ec6-bcef-c99ab521f111]
2017-04-19 20:49:33,102 INFO 13394 [celery.worker.strategy] strategy.py:55 - Received task: lms.djangoapps.grades.tasks.add[23a2da0c-75c4-4fa1-a716-9f60e7107d28]
2017-04-19 20:49:33,105 INFO 13394 [celery.worker.job] job.py:401 - Task lms.djangoapps.grades.tasks.add[23a2da0c-75c4-4fa1-a716-9f60e7107d28] succeeded in 0.00116802599223s: 12
2017-04-19 20:49:33,105 INFO 13394 [celery.worker.strategy] strategy.py:55 - Received task: lms.djangoapps.grades.tasks.add[8f3c8bc6-e586-444b-9f0b-87b5841bd49a]
2017-04-19 20:49:33,106 INFO 13394 [celery.worker.job] job.py:401 - Task lms.djangoapps.grades.tasks.add[d7248e00-4aee-4ec6-bcef-c99ab521f111] succeeded in 0.00304403700284s: 10
2017-04-19 20:49:33,107 INFO 13394 [celery.worker.strategy] strategy.py:55 - Received task: lms.djangoapps.grades.tasks.add[1a7813ab-ccda-4c0a-8bbb-8b980a93f706]
2017-04-19 20:49:33,109 INFO 13394 [celery.worker.job] job.py:401 - Task lms.djangoapps.grades.tasks.add[8f3c8bc6-e586-444b-9f0b-87b5841bd49a] succeeded in 0.00095349100593s: 14
2017-04-19 20:49:33,110 INFO 13394 [celery.worker.job] job.py:401 - Task lms.djangoapps.grades.tasks.add[1a7813ab-ccda-4c0a-8bbb-8b980a93f706] succeeded in 0.00122585699137s: 16
2017-04-19 20:49:33,111 INFO 13394 [celery.worker.strategy] strategy.py:55 - Received task: lms.djangoapps.grades.tasks.add[eeeb8275-ebd5-4019-b726-ab44ad324f35]
2017-04-19 20:49:33,113 INFO 13394 [celery.worker.job] job.py:401 - Task lms.djangoapps.grades.tasks.add[eeeb8275-ebd5-4019-b726-ab44ad324f35] succeeded in 0.000946536994888s: 18
```